### ISSUE TITLE
DECSYSTEM 2020 RH11-TM02/03-TU16/TU45/TU77 RELIABILITY DIAGNOSTIC(DSTUB) ACCL failure at Write Forward startup

### ISSUE THREAD

**Issue by jfcl** [KS10FPGA/KS10FPGA#25]
DSTUB Test 2 fails (and aborts) with a ACCL failure as follows:

```
*** TST # 2 ; TM03/TU77 0/0 ; ERR PC = 37004
*** RELIABILITY TEST - 1600BPI PE

ACCL NOT SET AT START OF OPERATION
 CS1     CS2     DS      ER      MR      AS   FC      DT      CK   TC
 004260  000100  010662  000000  004310  000  177300  142054  000  062000
DRBA = 000000    DRWC = 000000
DRCS2: INPUT READY,
DRCS1: DRIVE AVAILABLE, READY,
WRT FWD
DRDS:  MOL, DPR, DRY, PES, SDWN, BOT,
DRMR:  WRP1
DRTC:  PE-1600, COR DMP, ODD PAR
GOOD DATA WAS:  000000 000000 ,  BUFFER STARTED AT  00047000

TM03/TU77 0/0 ERRORS; ABORTING DRIVE FROM CURRENT PASS
```

**Comment by jfcl** [comment]
The error message in mentioned above is completely incorrect and misleading due to a latent software bug in the DSTUB diagnostic.

The actual failure is that an error condition was raised during the Write Forward command.   The specific failure is an implementation error in the IDB logic which raises a Non-Executable Function Error in the Error Register (MTER[NEF]).  The NEF Error causes the drive to assert Attention Active bit in the Drive Status Register (MTDS[ATA]),

Note: The software is actually checks the status of the MSB (ATA bit) of the Drive Status Register (MTDS).  The software is reporting the error as if the MSB (ACCL bit) of the Tape Control Register (MTTC) is asserted.

The code is question is a little obfuscated.   The code copies a tiny stub of timing software into the fast ACs and executes it.
The instruction at 040676 clearly copies a `RDIO D1, @MTDS` instruction into AC1.  The other code is copied by a `BLT` instruction.

**Comment by jfcl** [comment]
Fixed by 129db735194471ff5454e0aa69868baeb71cb798
